### PR TITLE
Update Keycloak DevConsole to use the authorization code grant by default

### PR DESCRIPTION
**Pull request by sberyozkin**
This PR updates KC dev console to use an `authorization_code` grant by default, `implicit` is not having a good reputation (it was easier to start with), and even though it is a test SPA, it still makes sense to use the code flow by default since it is recommended for SPAs/public clients and KC clients have to be configured to allow the implicit flow - hence a possible mismatch between the real realm configuration and what Dev Console for KC offers by default.

I was not changing `provider.html` card ids like  `implicitLoggedOut` to have diff ids depending on whether it was the implicit or code flow, these are internal ids, all works in both cases
 
Quarkus `web-app` itself uses it too.

### CODE REVIEW

File: docs/src/main/asciidoc/security-openid-connect-dev-services.adoc
Comment (gsmet) [inline docs/src/main/asciidoc/security-openid-connect-dev-services.adoc:50]:
```suggestion
If you set `quarkus.keycloak.devservices.grant.type=code` in `applicatin.properties` (this is a default value) then an `authorization_code` grant will be used to acquire both access and ID tokens. Using this grant is recommended to emulate a typical flow where a `Single Page Application` acquires the tokens and uses them to access Quarkus services.
```

File: docs/src/main/asciidoc/security-openid-connect-dev-services.adoc
Comment (gsmet) [inline docs/src/main/asciidoc/security-openid-connect-dev-services.adoc:66]: > (this is a default value)

I suppose it is not the default value anymore?

**Review by gsmet — DISMISSED** [review]

**Review by gsmet — COMMENTED** [review]
Doc looks good, I'll let @phillip-kruger approve the rest.

**Review by phillip-kruger — APPROVED** [review]
Nice. After this is merge I'll start looking into how we can add the swagger-ui requirements.

**Pull request merged**
